pages/tidal tools/analyze_database.md

```diff
@@ -57,18 +57,11 @@ For example, in Oracle databases, the Data Dictionary and AWR repository tables
   - password - A password for the corresponding user.
   - name - A common name for your database could be the same or different from db_name, but this value is arbitrary and only for your reference.
 
-### Oracle User
+### Oracle User Access
 
-You can either use the default `SYSDBA` user or you can create a user with the same set of permissions:
+You can can create a user with the [script and set of permissions defined in this script](/oracle_user.sql). You should provide a secure password on the first line.
```

Comment:
I don't know Oracle SQL too much but this looks good to me :)

Reply:
Thanks! can confirm, the script works.